documentation/modules/ref-iot-config-example.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-service-admin-guide.adoc
+
+[id='iotconfig-example-{context}']
+= Iot Config example
+
+This iot config example shows only the required options to create an `iotConfig`.
+
+[source,yaml,options="nowrap"]
+----
+include::../common/iot-config-default.yaml[]
+----
+<1> The infinispan service must be provided.
```

Comment:
```suggestion
<1> The Infinispan service must be provided.
```

Reply:
I think we might need a variable here: "Infinispan" becomes "Red Hat Data Grid" in downstream.

Reply:
@ctron good catch ! Updated